src/CoreBundle/UniversalUploader/Bridge/Chameleon/SaveToMediaLibraryService.php

```diff
@@ -102,7 +102,7 @@ public function saveUploadedFile(UploadedFileDataModel $uploadedFile)
                 'maxWidth' => $parameters->getMaxUploadWidth(),
                 'maxHeight' => $parameters->getMaxUploadHeight(),
             );
-            $uploadedFile->addErrorMessage($e->getMessage(), $e->getCode(), $context);
+            $uploadedFile->addErrorMessage($e->getMessage(), (int) $e->getCode(), $context);
```

Comment:
This one is a bit weird: Apparently, `getCode` can return `int|string` while the constructor only accepts `int`. This means that when rethrowing exceptions, the code has to be cast